### PR TITLE
ompgsql: In case of PG not available lets retry instead of dropping messages.

### PR DESCRIPTION
**Pull request by nickolai-dr**
Messages are lost in case of temporary PG operation errors, since `RS_RET_SUSPENDED` not propagated outside. 

Possibly related to https://github.com/rsyslog/rsyslog/issues/1105

### CODE REVIEW

**Comment by nickolai-dr** [comment]
Not sure how fixe code cov thing, my changes are not related to ElasticSearch...
```
============================================================================
See tests/test-suite.log
Please report to rsyslog@lists.adiscon.com
============================================================================
make[3]: *** [test-suite.log] Error 1
make[2]: *** [check-TESTS] Error 2
make[1]: *** [check-am] Error 2
make: *** [check-recursive] Error 1
make[3]: Leaving directory `/var/lib/buildbot/slave/rsyslog_ElasticSearch_codecov/build/tests'
make[2]: Leaving directory `/var/lib/buildbot/slave/rsyslog_ElasticSearch_codecov/build/tests'
make[1]: Leaving directory `/var/lib/buildbot/slave/rsyslog_ElasticSearch_codecov/build/tests'
program finished with exit code 2
```

File: plugins/ompgsql/ompgsql.c [hunk 1]
```diff
@@ -255,7 +255,8 @@ tryExec(uchar *pszCmd, wrkrInstanceData_t *pWrkrData)
 	pgRet = PQexec(pWrkrData->f_hpgsql, (char*)pszCmd);
 	execState = PQresultStatus(pgRet);
 	if (execState != PGRES_COMMAND_OK && execState != PGRES_TUPLES_OK) {
-		dbgprintf("postgres query execution failed: %s\n", PQresStatus(PQresultStatus(pgRet)));
+	// complain a lot in case any issues with DB communication
```
Comment (rgerhards) [inline plugins/ompgsql/ompgsql.c]: proper indention would be great

**Review by rgerhards — APPROVED** [review]

**Comment by rgerhards** [comment]
> Not sure how fixe code cov thing, my changes are not related to ElasticSearch...

This looks like a flake, I have re-initiated the run. I'll also enable the rest of the CI now. Note that there are some timing-related flakes. I'll watch out for them. If you see somthing not related to your patch, no need to worry (I'll take care).

Note that I am currently busy with other work and can probably not babysit CI until next week.

File: plugins/ompgsql/ompgsql.c (hunk 1 above)
Comment (nickolai-dr) [reply plugins/ompgsql/ompgsql.c]: Done. (artifact of my editor messing tab and spaces)

**Comment by nickolai-dr** [comment]
@rgerhards  can you please kick CI for `buildbot/rsyslog ElasticSearch codecov`, I do not have permissions to rerun build `Cannot rebuild: you need to have role 'admins'`. Any chance we can merge without it? Elastic thing seems like not related, this PR.

**Comment by nickolai-dr** [comment]
thanks!

**Pull request merged**